docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml

```diff
@@ -0,0 +1,82 @@
+@startuml
```

Comment:
Show explicit that is app2 is subsribed not another data, appropriate HMI request will be sent

Reply:
Fixed in   https://github.com/smartdevicelink/sdl_core_guides/pull/94/files